### ISSUE TITLE
Problems downloadig longer VODs (or segments)

### ISSUE THREAD

**Issue by MikeyOnline2** [MacielG1/VodRecovery#16]
Hello! 
I have problems downloading longer VODs or segments of the longer VODs (it's a subathon). (https://streamscharts.com/channels/lacy/streams/44766490571)

For some reason VodRecovery gives an error and can't download. In this case the VOD is published on the Twitch profile.
It doesn't matter if I use the option: 1-VOD Recovery, 1-Website Video Recovery, 1-Cunked (Best quality), 2-Specify Start & End times. Or 3-Download VOD (mp4), 3-From Twitch URL (Only for VODs or Highlights still up on Twitch), 1.Cunked (Best quality), 2-Specify Start & End times. The outcome is the same.
Normal vods give no problems

<img width="433" alt="2024-09-23_20-11-44" src="https://github.com/user-attachments/assets/981b2639-00c2-4732-946d-a1d5e19a0832">

<img width="303" alt="2024-09-23_20-13-36" src="https://github.com/user-attachments/assets/5db844d1-1ac4-4341-9fc1-de5d6076b9c7">

**Comment by MacielG1** [comment]
Thanks for reporting. I'll take a look into it

**Comment by MacielG1** [comment]
@MikeyOnline2 Check if the issue still occurs on the latest version: https://github.com/MacielG1/VodRecovery/releases/tag/1.3.1

**Comment by MikeyOnline2** [comment]
It seems to be solved!! I checked it with a random 48hours VOD. And it worked!

When I did a clone I got an error: "ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
yt-dlp 2024.8.6 requires requests<3,>=2.32.2, but you have requests 2.31.0 which is incompatible."

When I did it again, the error message was gone. So I think everything's okay.  Thank you!
